.github/workflows/linux-ci-helper.sh

```diff
@@ -183,6 +184,15 @@ elif [ "${CONTAINER_FULLNAME}" = "centos:centos7" ]; then
     INSTALL_CHECKER_PKGS="cppcheck jq"
     INSTALL_CHECKER_PKG_OPTIONS="--enablerepo=epel"
 
+    # [NOTE]
+    # To use C++11 std::get_time/put_time, you need GCC5 or higher.
+    # For building, we need to use devtoolset on CentOS 7.
+    # (devtoolset is not required in the s3fs execution.)
+    #
+    SCL_INSTALL=1
+    INSTALL_SCL_PACKAGES="centos-release-scl"
```

Comment:
One note about this, it could be we can't update package for EPEL7 (RHEL7, CentOS7, etc) anymore, as I think the EPEL building system will not allow me to use non EL7/EPEL7 packages (`devtoolset-11` is in the Software Collections SIG repository you are installing with `centos-release-scl`).

That said, I can still update the SPEC and README in my repository to explain this, so people can build the RPMs on their own.

The only limitation is that 1.93 will be the latest version available at EPEL7.

Reply:
I don't think we should break CentOS 7 in EPEL.  We can just revert the trivial C++11 change for now and un-revert it when CentOS 7 goes EOL in summer 2024.

Reply:
@juliogonzalez
Thank you for pointing out the packaging. That helps a lot!
I didn't even pay attention to the packaging.

@gaul
I agree.
I don't want to make changes that affect the package, so could you submit a PR to revert #2377?
I will confirm that PR and close this PR.

Reply:
@gaul I confirmed #2381 and merged it.